lib/utils/report.js

```diff
@@ -109,3 +109,26 @@ module.exports = function report(problem) {
 
 	result.warn(warningMessage, warningProperties);
 };
+
+/**
+ * @param {import('postcss').Node | undefined} node
+ * @param {number | undefined} index
+ * @param {number | undefined} endIndex
+ * @returns {number | undefined}
+ */
+function getStartLine(node, index, endIndex) {
+	if (node === undefined) return undefined;
+
+	// HACK: The `rangeBy` method has been added since PostCSS 8.4.0,
+	//       while the `node` may be produced by PostCSS 8.3.x or lower
+	//       that doesn't have the method.
+	//       See https://github.com/stylelint/stylelint/issues/5766
+	if (typeof node.rangeBy !== 'function') {
+		console.warn(
```

Comment:
We can link #5766 as reference at the same time?

And we can check whether it has been warned to ensure it's not noise.

Reply:
Make sense. Changed via 95a1ee6de. 👍🏼